app/views/govuk_publishing_components/components/docs/devolved_nations.yml

```diff
@@ -0,0 +1,77 @@
+name: Devolved Nations (experimental)
+description: A banner for linking to guidance for other nations.
+body: |
+  The component replaces uses of the important metadata component for guidance for other nations.
+
+  The component can display:
+
+  * nations that the guidance relates to
+  * a list of alternative URLs where applicable
+shared_accessibility_criteria:
+  - link
+accessibility_criteria: |
+  - Provide context for link text to highlight guidance is available.
+examples:
+  default:
+    data:
+      national_applicability:
+        england:
+          applicable: true
+  applies_to_two_nations:  
+    data:
+      national_applicability:
+        england:
+          applicable: true
+        wales:
+          applicable: true
+  applies_to_three_nations:  
+    data:
+      national_applicability:
+        england:
+          applicable: true
+        scotland:
+          applicable: true
+        wales:
+          applicable: true
+  applies_to_one_nation_individual_guidance_available:
+    data:
+      national_applicability:
+        england:
+          applicable: true
+        scotland:
+          applicable: false
+          alternative_url: /
+        wales:
+          applicable: false
+          alternative_url: /
+  applies_to_one_nation_individual_guidance_available:
+    data:
+      national_applicability:
+        england:
+          applicable: true
+        northern_ireland:
+          applicable: false
+          alternative_url: /
+        scotland:
+          applicable: false
+          alternative_url: /
+        wales:
+          applicable: false
+          alternative_url: /
+  applies_to_onesfsdfdsfdsdsfsdfds_nation_individual_guidance_available:
+    data:
+      national_applicability:
+        england:
+          applicable: false
+        northern_ireland:
+          applicable: true
+          alternative_url: /
+        scotland:
+          applicable: true
+          alternative_url: /
+  specific_heading level:
+    data:
```

Comment:
Suggestion: add a description, e.g.

`description: Use a different heading level for the main link title. Defaults to H2 if not passed.`